BLUESPAWN-client/headers/util/permissions/permissions.h

```diff
@@ -150,27 +177,48 @@ namespace Permissions {
 		* @param t An OwnerType containing value to be copied to otOwnerType
 		*/
 		Owner(IN const std::wstring& name, IN const bool& exists, IN const OwnerType& t);
```

Comment:
Values that can be passed in a register generally don't need to be const references.